controllers/packagebundle_controller.go

```diff
@@ -105,7 +105,32 @@ func (r *PackageBundleReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		if client.IgnoreNotFound(err) != nil {
 			return ctrl.Result{}, err
 		}
-		r.Log.Info("Bundle deleted (ignoring)", "bundle", req.NamespacedName)
+
+		// If the bundle controller detects that the active bundle is deleted,
+		// the bundle controller will validate the active bundle by namespace
+		// and name, and re-download the bundle.
+		nn, err := r.bundleManager.GetActiveBundleNamespacedName(ctx, r.Client)
+
+		if err != nil {
```

Comment:
```suggestion
		nn, err := r.bundleManager.GetActiveBundleNamespacedName(ctx, r.Client)
		if err != nil {
```
Nit: it's customary to keep the error checking immediately following a call. It helps us recognize upon scanning the pattern of "make a call, check the error".

Reply:
Noted. Thanks.